packages/react-error-overlay/src/__tests__/get-source-map.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @jest-environment jsdom
```

Comment:
you can stick this into the copyright comment block

```js
/**
 * Copyright (c) 2015-present, Facebook, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @jest-environment jsdom
 */
```

Reply:
That's better! I will do that too :)